src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -614,18 +617,15 @@ export default class AircraftInstanceModel {
      * @method isVisible
      */
     isVisible() {
-        // TODO: this if/else if would be cleaner with just if (this.flightPhase === FLIGHT_PHASE.WAITING) {}
         // hide aircraft on twys
         if (this.flightPhase === FLIGHT_PHASE.APRON || this.flightPhase === FLIGHT_PHASE.TAXI) {
             return false;
         }
 
         if (this.isTaxiing()) {
             // show only the first aircraft in the takeoff queue
-            const runway = this.fms.departureRunway;
-            const nextInRunwayQueue = runway.isAircraftNextInQueue(this);
-
-            return this.flightPhase === FLIGHT_PHASE.WAITING && nextInRunwayQueue;
+            return this.flightPhase === FLIGHT_PHASE.WAITING &&
```

Comment:
because you know I love to do things the hard way.  your way just makes so much more sense and I get to remove a `TODO`
👍